src/Examples/UnitModels/Operations/heat_exchanger_NTU_testing.ipynb

```diff
@@ -0,0 +1,513 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Heat Exchanger NTU Unit Model with Aqueous MEA System\n",
+    "\n",
+    "![](heat_exchanger_4.svg)\n",
+    "\n",
+    "**Problem Statement**: In this example, we will be transfering heat from hot to cold streams of MEA (monoethylamine), simulating heat integration of an aqueous MEA system.\n",
+    "\n",
+    "**Hot Side Inlet**\n",
+    "\n",
+    "Flow Rate = 60.54879 mol/s\n",
+    "\n",
+    "Mole fraction (CO2) = 0.0158\n",
+    "\n",
+    "Mole fraction (H2O) = 0.8747\n",
+    "\n",
+    "Mole fraction (MEA) = 0.1095\n",
+    "\n",
+    "Pressure = 202650 Pa\n",
+    "\n",
+    "Temperature = 392.23 K\n",
+    "\n",
+    "**Cold Side Inlet**\n",
+    "\n",
+    "Flow Rate = 63.01910 mol/s\n",
+    "\n",
+    "Mole fraction (CO2) = 0.0414\n",
+    "\n",
+    "Mole fraction (H2O) = 0.8509\n",
+    "\n",
+    "Mole fraction (MEA) = 0.1077\n",
+    "\n",
+    "Pressure = 202650 Pa\n",
+    "\n",
+    "Temperature = 326.36 K\n",
+    "\n",
+    "This example will demonstrate the simulation of the NTU heat exchanger by fixing the following degrees of freedom:\n",
+    "- heat transfer area\n",
+    "- heat transfer coefficient\n",
+    "- effectiveness\n",
+    "- hot and cold side pressure changes\n",
+    "\n",
+    "\n",
+    "IDAES documentation reference for heat exchanger 1D model: https://idaes-pse.readthedocs.io/en/latest/technical_specs/model_libraries/generic/unit_models/heat_exchanger_NTU.html\n",
+    "\n",
+    "This example utilizes the NTU heat exchanger for a shell and tube system. The IDAES library contains a more advanced `HeatExchangerLumpedCapacitance` model supporting a wall temperature and heat holdup for transident simulations; more details on the lumped capacitance heat exchanger may be found [here](https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/heat_exchanger_lc.html)."
```

Comment:
I am not sure we should reference the Lumped Capacitance model here - they are not really related (the LC model is an extension of the 0D model).